### PR TITLE
misc/metrics: Use correct event name for dcutr metrics

### PR DESCRIPTION
**Pull request by dignifiedquire**
Currently when both the `relay` and `dcutr` features are enabled this generates an error.

# Description

Currently when both the `relay` and `dcutr` features are enabled this generates an error.

## Change checklist

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] A changelog entry has been made in the appropriate crates

### CODE REVIEW

**Review by mxinden — COMMENTED** [review]
Good catch. Thanks @dignifiedquire!

Mind bumping `libp2p-metrics` to `v0.6.1` and adding a changelog entry to `misc/metrics/CHANGELOG.md`? I can cut a patch release once merged.

**Comment by dignifiedquire** [comment]
done

File: misc/metrics/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+
```
Comment (mxinden) [inline misc/metrics/CHANGELOG.md]:
```suggestion
```

**Review by mxinden — APPROVED** [review]
Thanks @dignifiedquire!

**Comment by mxinden** [comment]
[Tagged](https://github.com/libp2p/rust-libp2p/releases/tag/libp2p-metrics-0.6.1) and [published](https://crates.io/crates/libp2p-metrics/0.6.1).

**Pull request merged**